client/home/lib/welcome/welcomemodal.coffee

```diff
@@ -31,32 +31,33 @@ module.exports = class WelcomeModal extends kd.ModalView
   showCongratulationModal: ->
 
     { appStorageController } = kd.singletons
-
     appStorage = appStorageController.storage "WelcomeSteps-#{globals.currentGroup.slug}"
-    appStorage.fetchValue 'OnboardingSuccessModalShown', (result) ->
```

Comment:
fetchValue is not working at will. so decided to get `OnboardingSuccessModalShown` attribute from appStorage??